shell.nix

```diff
@@ -0,0 +1,24 @@
+{ pkgs ? import <nixpkgs> {}
+, buildNodeJs ? pkgs.callPackage <nixpkgs/pkgs/development/web/nodejs/nodejs.nix> { python = pkgs.python3; }
```

Comment:
can this on it's own line so it's easier to read? i don't know about any nix linters but when they are all on one line i get scarwed

Reply:
I dont know what you are saying here, this is the standard formatting of nix, putting each part of the attrset arguement to the function on a new line with leadings comma